cmd/tracee-ebpf/main.go

```diff
@@ -209,54 +217,56 @@ func main() {
 			&cli.StringSliceFlag{
 				Name:    "trace",
 				Aliases: []string{"t"},
-				Value:   nil,
+				Value:   cli.NewStringSlice(config.Config.Trace...),
 				Usage:   "select events to trace by defining trace expressions. run '--trace help' for more info.",
 			},
```

Comment:
there's no point in having the CLI flags here and the config file there. If we're going with this approach than the Flag structs should move to the config package as well (created by the package and returned to be used here)

Reply:
@mtcherni95 suggested we don't use a global instance for the config, which for the current implementation I also think would be better. In that case we'd need to change the flags to be returned from a function which receives a config as a parameter (something like `GetCliFlags(config CliConfig) ([]cli.Flag)`. Either that or we keep them here and I pass the return value from the returned config as the default value instead.